server/src/main/java/au/org/aodn/ogcapi/server/core/mapper/Converter.java

```diff
@@ -98,6 +98,10 @@ default <D extends StacCollectionModel> Collection getCollection(D m, String hos
             collection.getProperties().put(CollectionProperty.STATUS, m.getSummaries().getStatus());
         }
 
+        if (m.getSummaries() != null && m.getSummaries().getCredits() != null) {
+            collection.getProperties().put(CollectionProperty.CREDITS, m.getSummaries().getCredits());
```

Comment:
`m.getSummaries().getCredits()` might need to have null check as well, `m.getSummaries() != null` doesn't mean `m.getSummaries().getCredits() != null`

Reply:
Sorry Viet i didn't get your point. Did you mean anywhere else? Because the code you mentioned already has null check for credits
![image](https://github.com/aodn/ogcapi-java/assets/102270719/0b4d5475-8f90-4a73-9d35-39fad911641a)

Reply:
Sorry I overlooked that bit! All good.